Tasks/google-play-release/GooglePlay.ts

```diff
@@ -816,6 +840,18 @@ function getVersionCodeListInput(): number[] {
     }
 }
 
+function shouldPickObbForApk(apk: string, mainApk: string): boolean {
+    const shouldPickObbFile = tl.getBoolInput('shouldPickObbFile', false);
```

Comment:
It's not a good practice to put 'input reading' methods in a separate function from all the others.  This does make things more readable in this case, but possibly at the cost of unintended side effects.    I think it would be better to read these values with the other `getBoolInput` calls and pass the values in as parameters.